xml/System.Net.Http/HttpClient.xml

```diff
@@ -363,9 +363,36 @@ public class GoodController : ApiController
         <ReturnType>System.Net.IWebProxy</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets the global Http proxy.</summary>
+        <value>A proxy used by every call that instantiates a <see cref="T:System.Net.Http.HttpWebRequest" />.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property gets or sets the global proxy. The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property determines the default proxy that all <xref:System.Net.Http.HttpClient> instances use if no proxy is set explicitly in the <xref:System.Net.Http.HttpClientHandler> passed through its constructor. 
+
+If the <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property is overridden, all subsequent instances of the <xref:System.Net.Http.HttpClient> class that does not define its own Proxy in <xref:System.Net.Http.HttpClientHandler> will be using the new defined Proxy in <xref:System.Net.Http.HttpClient.DefaultProxy%2A>.
+
+The <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property reads proxy settings from your system configuration and follows a certain behavior depending on your platform: 
```

Comment:
I don't think we need to document platform-specific behavior here. Can we generalize?

```suggestion
The default instance returned by the <xref:System.Net.Http.HttpClient.DefaultProxy%2A> property will be specified based on environment variables, and if those are not defined, the user's proxy settings.
```

Reply:
@scalablecory can you elaborate? The result is slightly different depending on the platform, so I think it's great to have those differences explained.

Reply:
@carlossanlop it looks like the platform-specific behavior is simply *where* the system-wide proxy is configured. I don't know if that's worth documenting here.

However, on second review, if Linux does not have a standard system proxy configuration and so we simply don't try, that specifically seems worth documenting.

Reply:
Yep, agreed. I think it's better to have this documented so we make life easier for Unix users.
No one likes going into the rabbit hole in StackOverflow, wondering why something is not working and not finding any answers.😄

Reply:
I reworded this a little bit on my last commit.